lotus/sem_ops/sem_topk.py

```diff
@@ -180,6 +186,14 @@ def llm_quicksort(
     stats = {}
     stats["total_tokens"] = 0
     stats["total_llm_calls"] = 0
+    if safe_mode:
+        sample_prompt = get_match_prompt_binary(docs[0], docs[1], user_instruction, strategy=strategy)
+        estimated_quickselect_calls = 2 * K
+        estimated_quicksort_calls = 2 * len(docs) * np.log(len(docs))
+        estimated_total_calls = estimated_quickselect_calls + estimated_quicksort_calls
+        estimated_total_tokens = lotus.settings.lm.count_tokens(sample_prompt) * estimated_total_calls
+        print("Quicksort:")
```

Comment:
Let's remove the `Quicksort` print, and similarly the `Heap Sort` print.

Reply:
@sidjha1  Naive sort as well?

Reply:
Yup